plugins/acss-kit/assets/foundation/sass/_globals.scss

```
@@ -0,0 +1,167 @@
+:root {
+  --font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Helvetica Neue",
```

Stylelint (17.10.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)